### PR TITLE
Resolved the issue in datalake/file where file/path/directory/share/f…

### PR DESCRIPTION
**Pull request by katmsft**
…ilesystem name are not encoded.

### CODE REVIEW

File: sdk/storage/azure-storage-files-shares/test/share_client_test.cpp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: MIT
 
 #include "share_client_test.hpp"
+#include "azure/storage/common/crypt.hpp"
```
Comment (Jinming-Hu) [inline sdk/storage/azure-storage-files-shares/test/share_client_test.cpp:5]: reformat

**Review by Jinming-Hu — APPROVED** [review]

File: sdk/storage/azure-storage-files-shares/test/share_client_test.cpp (hunk 1 above)
Comment (katmsft) [reply sdk/storage/azure-storage-files-shares/test/share_client_test.cpp:4]: Don't know why my Clang formatter does not catch this one.

**Pull request merged**